pkg/controller/operator/seed/reconciler_test.go

```diff
@@ -279,6 +293,64 @@ func TestBasicReconciling(t *testing.T) {
 				return nil
 			},
 		},
+
+		{
+			name:            "nodeport-proxy annotations are carried over to the loadbalancer service",
+			seedToReconcile: "seed-with-nodeport-proxy-annotations",
+			configuration: &operatorv1alpha1.KubermaticConfiguration{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test",
+					Namespace: "kubermatic",
+				},
+				Spec: operatorv1alpha1.KubermaticConfigurationSpec{
+					Ingress: operatorv1alpha1.KubermaticIngressConfiguration{
+						Domain: "example.com",
+					},
+				},
+			},
+			seedsOnMaster: []string{"seed-with-nodeport-proxy-annotations"},
+			syncedSeeds:   sets.NewString("seed-with-nodeport-proxy-annotations"),
+			assertion: func(test *testcase, reconciler *Reconciler) error {
+				if err := reconciler.reconcile(reconciler.log, test.seedToReconcile); err != nil {
+					return fmt.Errorf("reconciliation failed: %v", err)
+				}
+
+				seedClient := reconciler.seedClients["seed-with-nodeport-proxy-annotations"]
+
+				seed := kubermaticv1.Seed{}
+				if err := seedClient.Get(reconciler.ctx, types.NamespacedName{
+					Namespace: "kubermatic",
+					Name:      "seed-with-nodeport-proxy-annotations",
+				}, &seed); err != nil {
+					return fmt.Errorf("failed to retrieve Seed: %v", err)
+				}
+
+				if !kubernetes.HasFinalizer(&seed, common.CleanupFinalizer) {
+					return fmt.Errorf("Seed copy in seed cluster does not have cleanup finalizer %q", common.CleanupFinalizer)
+				}
```

Comment:
All of this can be removed, the point of this test is not to validate finalizers.

Reply:
will do, thanks.